src/components/Sidebar/Sidebar.js

```diff
@@ -141,67 +153,76 @@ export class Sidebar extends PureComponent<Props, State> {
-              <ScrollbarOnlyVertical
-                autoHide
-                renderThumbVertical={({ style, ...props }) => (
-                  // Info: Styled-components not working here yet
-                  //       --> PR 286 @ react-custom-scrollbars will fix this
-                  <div
-                    {...props}
-                    style={{
-                      ...style,
-                      background: COLORS.transparentWhite[700],
-                      borderRadius: '6px',
-                    }}
-                    className="thumb-vertical"
-                  />
-                )}
-                renderTrackHorizontal={props => (
-                  <div
-                    {...props}
-                    style={{
-                      display: 'none',
-                    }}
-                  />
-                )}
-              >
-                <IntroductionBlurb
-                  isVisible={!finishedOnboarding && introSequenceStepIndex >= 1}
-                />
-                <Projects offset={`${interpolated.firstProjectPosition}px`}>
-                  {projects.map(project => (
-                    <Fragment key={project.id}>
-                      <Tooltip
-                        title={project.name}
-                        position="right"
-                        transitionFlip={false}
-                      >
-                        <SidebarProjectIcon
-                          size={SIDEBAR_ICON_SIZE}
-                          id={project.id}
-                          name={project.name}
-                          color={project.color}
-                          iconSrc={project.icon}
-                          isSelected={
-                            finishedOnboarding &&
-                            project.id === selectedProjectId
-                          }
-                          handleSelect={() => selectProject(project.id)}
-                        />
-                      </Tooltip>
-                      <Spacer size={18} />
-                    </Fragment>
-                  ))}
-                  <AddProjectButton
-                    size={SIDEBAR_ICON_SIZE}
-                    onClick={createNewProjectStart}
-                    isVisible={
-                      finishedOnboarding || introSequenceStepIndex >= 2
-                    }
-                    isOnline={isOnline}
-                  />
-                </Projects>
-              </ScrollbarOnlyVertical>
+              <IntroductionBlurb
+                isVisible={!finishedOnboarding && introSequenceStepIndex >= 1}
+              />
+              <DragDropContext onDragEnd={this.onDragEnd}>
+                <Scrollbars autoHide>
+                  <Droppable droppableId="droppable">
+                    {provided => (
+                      <div ref={provided.innerRef}>
+                        <Projects
+                          offset={`${interpolated.firstProjectPosition}px`}
```

Comment:
@melanieseltzer I think I've found the cause of the offset issue during dragging. By commenting the offset prop for `Projects` the drag offset issue is not present but the drag behaviour is not working as expected. (Dragging is working but after releasing the icon is sliding horizontally.)
Not sure how to fix this.